nzedb/Contents.php

```diff
@@ -128,34 +154,54 @@ public function getByID($id, $role)
 		return $this->row2Object($row);
 	}
 
+	/**
+	 * @param $content
+	 *
+	 * @return mixed
```

Comment:
Don't think 'mixed' is correct. Looks like it is an object of some kind, which means the @param can be updated too